src/test/java/com/github/dockerjava/core/command/ExecStartCmdImplTest.java

```diff
@@ -92,4 +90,25 @@ public void execStartAttached() throws Exception {
         assertNotNull(responseAsString);
         assertTrue(responseAsString.length() > 0);
     }
+    
+    @Test(groups = "ignoreInCircleCi", expectedExceptions = NotFoundException.class)
```

Comment:
Things are not exactly clear here I admit and now I'm scratching my head wondering why I did things this way. Let me clean this up a bit along with the other minor changes.